src/CardWrapper.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import ReactDOM, {unstable_renderSubtreeIntoContainer as renderSubtreeIntoContainer} from 'react-dom'
+import assign from 'object-assign'
+import Card from '../src/Card'
+import '../src/styles/main.scss';
+
+const defaultProps = {
+    active: false,
+    group: 'main',
+    tooltipTimeout: 500
+  };
+
+const propTypes = {
+    parent: PropTypes.string.isRequired,
+    active: PropTypes.bool,
+    group: PropTypes.string,
+    tooltipTimeout: PropTypes.number,
+    className:PropTypes.string
+};
+
+var portalNodes = {};
+
+class CardWrapper extends Component { 
```

Comment:
Not sure how important is it but I think we should name this one `Tooltip` because it basically replaces the previous `Tooltip` class

Reply:
I didn't get it ?!

Reply:
I mean in https://github.com/romainberger/react-portal-tooltip/blob/master/src/index.js we have a class called `ToolTip` which does exactly what your `CardWrapper` does. Therefore I think we should rename `CardWrapper` to `ToolTip`